src/mesh/mod.rs

```diff
@@ -189,59 +208,79 @@ impl ShapeMesh {
         camera: &Camera,
         env: &Environment,
     ) -> Result<()> {
-        // Visit each node of the tree
+        // Visit each node of the tree.
         // TODO: we might want visit the nodes in a different order (see #16)
 
-        let focus_point = self.get_focus(camera);
-
         let it = self.tree.iter()
             .filter_map(|n| n.leaf_data().map(|data| (data, n.span())));
         for (leaf_data, span) in it {
             match leaf_data {
-                // If there is a view available, render it
+                // If there is a view available, render it.
                 &MeshStatus::Ready(ref view) |
                 &MeshStatus::Requested { old_view: Some(ref view) } => {
                     view.draw(surface, camera, env, &self.renderer)?;
+                    // TODO: Debug mode can be removed soon.
```

Comment:
Mh... no, I think I want to keep that :P We still want to see the octree structure; we're just not interested in highlighting the cube we're looking at.